.env.example

```diff
@@ -1,7 +1,47 @@
-DEBUG=
+# Unused atm (jest sets this to `test`)
 NODE_ENV=development
-TMPDIR=$TMPDIR
-#To allow testing different executables in the bin tests
-#PK_TEST_DOCKER_IMAGE=$image #Specify the docker image that the `docker-run.sh` uses
-#PK_TEST_COMMAND=scripts/docker-run.sh #Specify the executable we want to test against
-#PK_TEST_COMMAND_DOCKER=DOCKER #Specify if the test is for docker for filtering relevant tests
+
+# Debug node modules - https://nodejs.org/api/cli.html#node_debugmodule
+# NODE_DEBUG=
+
+# Debug node native modules - https://nodejs.org/api/cli.html#node_debug_nativemodule
+# NODE_DEBUG_NATIVE=
+
+# Path to PK executable to override tests/bin target
+# PK_TEST_COMMAND=
+
+# If set, indicates that `PK_TEST_COMMAND` is targetting docker
+# PK_TEST_COMMAND_DOCKER=
+# Login credentials for GitLab's container registry
+# Automatically injected from CI/CD
+CI_REGISTRY_IMAGE=registry.gitlab.com/matrixai/open-source/js-polykey
+CI_REGISTRY_USER=
+CI_REGISTRY_PASSWORD=
```

Comment:
I'm standardising these variables as what we use because that's what gitlab offers as auto-injected variables.